front/src/applications/stdcm/utils/consistValidation.ts

```diff
@@ -0,0 +1,66 @@
+import { kgToT } from 'utils/physics';
+
+export const validateTotalMass = (
+  tractionEngineMass?: number,
+  towedMass?: number,
+  totalMass?: number
+) => {
+  if (!totalMass) {
```

Comment:
```suggestion
  if (!totalMass || !tractionEngineMass) {
```
I think we should also check that at the beginning, then you won't have to check it again or give it a default value :)

Reply:
Shouldn’t this function still verify the value of the mass even if tractionEngineMass isn’t provided?